packages/backend/src/index.ts

```diff
@@ -72,7 +79,16 @@ function makeCreateEnv(config: Config) {
     const logger = root.child({ type: 'plugin', plugin });
     const database = databaseManager.forPlugin(plugin);
     const cache = cacheManager.forPlugin(plugin);
-    return { logger, cache, database, config, reader, discovery, tokenManager };
+    return {
+      logger,
+      cache,
+      database,
+      config,
+      reader,
+      discovery,
+      tokenManager,
```

Comment:
No need for a `forClient` for this one? Just checking. It's sometimes a good pattern if you read per-plugin info/config, or log or emit metrics that are per plugin, etc.

Reply:
Don't think we have a use case for that right now, but I'm also not very familiar with the current pattern. Cc @mtlewis for additional thoughts.

Reply:
Alright!